src/components/StationsListItem.jsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styles from './StationsListItem.module.scss';
 import Button from './Button';
-import { ReactComponent as playLarge } from '../images/play-large.svg';
+import { ReactComponent as playMediumIcon } from '../images/play-medium.svg';
```

Comment:
It's generally a good practice to "group" your variable names by nouns and adjectives - in this case I would use something like "[noun][noun][adjetive]" = "playIconMedium" or "[adjective][noun][noun]" = "mediumPlayIcon"